docs/nginx.md

```diff
@@ -109,10 +115,11 @@ server {
     proxy_read_timeout                  900;
   }
 }
+}
 EOF
 
 # Now, create a password file for "testuser" and "testpassword"
-docker run --entrypoint htpasswd httpd:2.4 -bn testuser testpassword > auth/nginx.htpasswd
+docker run --rm --entrypoint htpasswd  registry:2 -bn testuser testpassword > auth/nginx.htpasswd
```

Comment:
Does this not create a dependency on the registry image having `htpasswd`? I didn't even know it was there...